### DISCUSSION TITLE
AdMob and Unity

### DISCUSSION THREAD

**Discussion by Winamp9** [EMI-INDO/emi-indo-cordova-plugin-admob#12] [General]
Is it possible to connect an Admob plugin and a separate Unity plugin for users from Russia in one application? Will it work?

**Comment by EMI-INDO** [comment]
@Winamp9 

it's best to ask Admob and Unity directly

https://forum.unity.com/threads/can-we-use-both-unity-ads-and-admob-in-one-project.956427/

**Comment by Winamp9** [comment]
EMI-INDO man, how do you monetize users from Russia?

**Comment by EMI-INDO** [comment]
@Winamp9 

It's best to ask directly on the unity forum
https://forum.unity.com/

I currently use admob ads, I don't use unity ads.
